src/test/mock.ts

```diff
@@ -64,7 +64,8 @@ export const MOCK_QUESTION: Question = {
     token_value: 5,
     type_name: '',
     visible_distractor_count: 1,
-    is_author: true
+    is_author: true,
+    is_checkbox: false
```

Comment:
does it give you an error if you add an extra `,` at the end? If not please add it.